python/lsst/cp/pipe/defects.py

```diff
@@ -74,6 +77,23 @@ class MeasureDefectsTaskConfig(pipeBase.PipelineTaskConfig,
     """Configuration for measuring defects from a list of exposures
     """
 
+    thresholdType = pexConfig.ChoiceField(
+        dtype=str,
+        doc=("Defects threshold type: 'STDEV' or `VALUE`."),
+        default='STDEV',
+        allowed={'STDEV': "Use number of sigma given standard deviation.",
+                 'VALUE': "Use pixel value."},
+    )
+    darkCurrentThreshold = pexConfig.Field(
+        dtype=float,
+        doc="Dark current threshold (in e-/sec) to define hot/bright pixels in dark images.",
```

Comment:
Is this only applicable if `thresholdType='Value'`?  If so it should be in the docstring.